### ISSUE TITLE
Setup GitHub Action for CI on Cloud Platform

### ISSUE THREAD

**Issue by jvillemare** [BITNULLS/storybook#22]
**Due Date:** Sprint 2 or earlier

**Description:**
This issue is blocked by #19.  Once the Cloud Platforms are setup, it would be nice to have our GitHub repo deployed in a continuous integration (CI) to our Cloud Platform so that we don't have to manually do it each time.

**Objectives:**
 - [ ] Find or come up with `.yaml` script to deploy GitHub repo to target cloud platform.

**JIRA issue number:** To be filled in by @vivatheest
